packages/waku/src/vite-rsc/lib/handler.ts

```diff
@@ -19,8 +19,6 @@ type HandleRequestOutput = Awaited<ReturnType<HandleRequest>>;
 
 // cf. `handler` in packages/waku/src/lib/middleware/handler.ts
 export async function handleRequest(ctx: HandlerContext) {
-  await import('virtual:vite-rsc-waku/set-platform-data');
```

Comment:
Do we drop the "setPlatformData" workaround completely?

Reply:
Yes, this was only for passing fs router "files" from build time to production runtime, so it can be removed.

Reply:
Alright, this is related with the above discussion.

I'm wondering how the Waku minimal API can expose some capabilities for the build process, so that libraries can use. (I would like to allow other developers to build a replacement for `waku/router` as a separate library.)

I wasn't super happy with `setPlatformData`, so happy to drop it for now, but I guess eventually we need such capabilities for library authors.

Reply:
Maybe because `define-router` only used it for caching? If no platform data, it may only fallback to compute it again during runtime for the first time.